lib/covenants/rules.js

```diff
@@ -528,6 +528,16 @@ rules.countOpens = function countOpens(tx) {
   return total;
 };
 
+/**
+ * Check whether this covenant counts against the open limit.
+ * @param {String} type
+ * @returns {Boolean}
+ */
+
+rules.isOpenLimited = function isOpenLimited(typeStr) {
```

Comment:
Nit: no type hints in variable name. Generally `type` is a `Number` and `val` or `value` is the `String` - see `types` and `typesByVal` objects